### DISCUSSION TITLE
Feedback on the "Basic anatomy" tutorial

### DISCUSSION THREAD

**Discussion by konnov** [informalsystems/quint#595] [Polls]
We are experimenting with different forms of tutorials for Quint. If you have finished the tutorial, please let us know, which kind of the tutorial you like better: the CodeTour tutorial, the Markdown tutorial, or both.

If you have any further suggestions about improving the tutorial, please leave a comment below.

**Comment by pdini** [comment]
In the Codetour tutorial, if I want to see the Codetour text of a previous step, say Step 5, and I am at Step 12, I need to step back seven times to get to it. It would be nice to be able to see the Step 5 text just by hovering over it (like a tooltip), without leaving Step 12.

The exercises were great, very helpful. Overall I feel I learned something that is likely to stick. Nice work!

**Reply by pdini** [reply]
Today I noticed that what I ask for above is already available. One just needs to hover the mouse over the desired step in the navigator in the left panel of VS Code and the text from that step pops out. Great! :)

**Comment by BMASHRA** [comment]
CodeTour tutorial is great for beginners especially. step by step instructions are thorough and makes it easy to follow. Very helpful. Thanks

**Comment by nsengupta** [comment]
I think a tutorial that (1) takes one through exact stages (post-installation) that a Spec-writer is advised to follow and (b) provides examples (the facility of associating questions is great :clap: !) which become difficult gradually, will be very welcome addition/refactoring. For example, I think introducing _tic-tac-toe_ just after _hello_ - instead of _coin_ -  will make the journey, much smoother. 

- Describe a summary of what is it that we are going to build ( 'O' should never win: yes, but why is that important?)
- Guide how to write the Quint specification of the summary above 
- Instruct how to run and what to expect (with each additional change/addition to the spec)
- Provide hints how to carry forward this knowledge to apply to other similar problems (exercises, yes!)

My 2 cents.
